common-ui/src/main/res/values/themes.xml

```diff
@@ -143,6 +143,9 @@
 
         <item name="tooltipPanelBackgroundColor">@color/tooltipPanelBackgroundColorDark</item>
         <item name="alertPanelBackgroundColor">@color/alertPanelBackgroundColorDark</item>
+
+        <item name="autofillDialogTitleColor">@color/white</item>
```

Comment:
It's a bit unfortunate that we had to define a special color for this since it's a pretty standard color. We could probably just reuse normalTextColor here. But I understand that this is more of a design system problem.


Same goes for the explanation color. It could almost be secondaryTextColor.